index.html

```diff
@@ -0,0 +1,150 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+<style>
+body {
+    font-family: 'Poppins';
+}
+</style>
+    </head>
+    <body>
+      <nav>
+        <ul>
+          <li><a class="logolink" href="">My Logo</a></li>
+          <li style="float: right;"><a href=""><img src="hammenu.png" alt="ham" style="width: 45px;height: 45px;"></a></li>
+        </ul>
+      </nav>
+        <header>
+            <div class="textdiv"><h1>I’m Alexandra <br>
+            Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look <br> through 
+              some of my work and experience! If you like what you see and have a project <br>
+              you need coded, don’t hestiate to contact me.</p>
+              <p style="color: #7F8CFF;">LET’S CONNECT</p>
+            <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section>
+            </div>
+        </header>
+       <div class="container">
+        <div class="imgdiv">
+            <img src="image1.png" alt="Nature" height="300" width="300">
+        </div>
+        <div class="tonic">
+                <h1 class="popheading">Tonic</h1>
+                <p class="popp13">CANOPY * <span class="popp13">Back End Dev</span> * <span class="popp13">2015</span></p>
+                <br>
+                <p class="popp15">A daily selection of privately personalized reads; no accounts <br> or sign-ups required.</p>
```

Comment:
your project cards are appearing different from the design, kindly ensure you are adhering strictly to the design😉
your project
<img width="258" alt="yourApp" src="https://user-images.githubusercontent.com/64643995/212498591-0e109cfa-c5c7-4fa2-9e88-4ea5163b8c30.png">

the design
<img width="187" alt="the-design" src="https://user-images.githubusercontent.com/64643995/212498609-c6027974-79a5-465c-b026-757369a4a5f4.png">